pandas/tests/plotting/test_converter.py

```diff
@@ -373,3 +373,22 @@ def test_convert_nested(self):
         r1 = self.pc.convert([data, data], None, self.axis)
         r2 = [self.pc.convert(data, None, self.axis) for _ in range(2)]
         assert r1 == r2
+
+
+class TestTimeDeltaConverter:
+    def setup_method(self, method):
```

Comment:
We're trying to move away from the `setup_`-style of tests. Can you use the full name in the test?

Reply:
you mean `timedelta_converter_method`? @TomAugspurger